dev-requirements.txt

```diff
@@ -0,0 +1,9 @@
+requests>=1.0.0
```

Comment:
Change `>=` to `==` (fix version) for package requirements (`requests`, `setuptools`, `wheel` and `twine`).
To trigger `dependabot` to create a new PR for updating dependencies.

Reply:
@AHReccese  Would you please re-check this one? (all basic requirements not only `requests`)

Reply:
my bad, you are right, hold my beer & lemi apply this.

Reply:
@sepandhaghighi 
Applied sir.